xsec/binningStudy.C

```diff
@@ -10,9 +10,9 @@
 #include "includes/SignalDefinition.h"
```

Comment:
Not related to adler angle.

Could go directly into master. Or into another branch.